runtime/src/partitioned_rewards.rs

```diff
@@ -47,6 +47,10 @@ pub enum TestPartitionedEpochRewards {
     None,
     CompareResults,
     ForcePartitionedEpochRewardsInOneBlock,
+    PartitionedEpochRewardsConfigRewardBlocks {
```

Comment:
nobody is using this. I assume you plan to have someone use this?

Reply:
yeah. it will be used in the following PR
https://github.com/solana-labs/solana/pull/32415

Reply:
To be clear this is split from 32415. 
If you prefer, 32415 has the complete code to review..